airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+
+        # add event id to the list
+        structure['ids'].append(events[1].id)
+        # send list to the browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify it was updated
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": [events[1].id]})
+
+        # add another event id to the list
+        structure['ids'].append(events[2].id)
+        # send list to the browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify it was updated
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": [events[1].id, events[2].id]})
+
+        # remove an event id from the list
+        structure['ids'].remove(events[1].id)
+        # send list to the browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify event id was removed
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": [events[2].id]})
+
+
+    def test_invalid_starred_event_id(self):
+
+        url = self.url
+        events = self.events
+
+        # pass url to the browser
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        # get the empty list of event ids in json format 
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": []})
+
+        event = Event.objects.get(title='Test event')
+        new_event = Event.objects.create(
+            title='The New Event',
+            slug='event4',
+            description=event.description,
+            start_time=event.start_time,
+            privacy=Event.PRIVACY_PUBLIC,
+            placeholder_img=event.placeholder_img,
+            location=event.location,
+        )
+        # add event id to the list
+        structure['ids'].append(new_event.id)
+        # send list to the browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify it was updated
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": [new_event.id]})
+
+        # delete event
+        new_event = new_event.delete()
+        # send updated list to the browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify it was updated
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": []})
+
+
+    def test_anonymous_user(self):
+        # log out user from setUp()
+        self.client.logout()
```

Comment:
I was just starting to write this test when I pushed my branch for you to look at.